eng/docgeneration/exclude-doc-index.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I'm not sure I like this method of excluding. Do you know why these are excluded? Do they not have docs? If that is the case then your JS fix should just hide them and we should be OK.

Reply:
Two parts of the files: services and artifacts.

I am not quite sure why we exclude the artifacts, it is originally there. I don't want to lose the ability of excluding artifacts. 

For exclude services, in Java, I did not see boms and parents get displayed in azure.github.io. However, if we parsing ci.yml, we will generate the parents.html and boms.html. Since no versioned docs in these two pages, we display nothing. However, as they are docfx generated files(toc.html, parents.html, boms.html), we are not able to use JS file to hide them from toc.html since we are not able to preload and check whether there are versioned doc exists